tests/test_rasterdefinition.py

```diff
@@ -379,3 +413,47 @@ def test_Rasterdefinition_DeleteRasterDefinition_Type_Wrong_RasterDefinitionID(s
             expectedType = expectedType,
             checkForData = False
         )
+
+
+# ========================================== Private Methods ==========================================
```

Comment:
I think my naming convention fitting into what's written there:
There should be no difference between method names, as there are no truly private methods in python. Not like in C# or Java.
[Class Names](https://peps.python.org/pep-0008/#class-names)
Class names should normally use the CapWords convention.

The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.

Note that there is a separate convention for builtin names: most builtin names are single words (or two words run together), with the CapWords convention used only for exception names and builtin constants.

Reply:
Isn't CapWords/CamelCase utilized for class naming? Also, the `_` prefix convention seems like it would fit here:

> In addition, the following special forms using leading or trailing underscores are recognized (these can generally be combined with any case convention):
> 
>   `_single_leading_underscore`: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.

Reply:
ok, Added _ before the method names, but still using the CapWords to signal it's a method not a variable

Reply:
Why change your mind on naming conventions? Should I redesign all the methods for Snake case then? It would look unprofessional if some methods are in that format and some are not.
What advantage Snake_Case have over CapWords/CamelCase to make the need to redesign the whole naming convention for the repository?

Reply:
It looks like the other methods follow a hybrid snakecase/camel/capwords; I'm only asking that the private methods follow snakecase so it falls in line with PEP8. Other methods do not need to be changed

Example of hybrid casing:
```test_Rasterdefinition_GetRasterDefinitionsForCurrentUser_Filter_Resolution_500```